### PR TITLE
Add Scala Play Guide

### PR DESCRIPTION
**Pull request by unicodeveloper**
Here is a comprehensive guide to deploy Scala Framework to Railway

CleanShot 2024-11-08 at 13 47 01@2x

### CODE REVIEW

File: src/docs/guides/play.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+3. [Using the CLI](#deploy-from-the-cli).
+4. [Using a Dockerfile](#use-a-dockerfile).
+
+Now, let's create a Play app!
+
+## Create a Play App
+
+**Note:** If you already have a Play app locally or on GitHub, you can skip this step and go straight to the [Deploy Play App to Railway](#deploy-play-app-to-railway).
+
+To create a new Scala Play app, ensure that you have [JDK](https://www.oracle.com/java/technologies/downloads/) and [sbt](https://https://www.scala-sbt.org/download/) installed on your machine.
```
Comment (brody192) [inline src/docs/guides/play.md:20]: https://https://

File: src/docs/guides/play.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+1. [One-click deploy from a template](#one-click-deploy-from-a-template).
+2. [From a GitHub repository](#deploy-from-a-github-repo).
+3. [Using the CLI](#deploy-from-the-cli).
+4. [Using a Dockerfile](#use-a-dockerfile).
+
+Now, let's create a Play app!
+
+## Create a Play App
+
+**Note:** If you already have a Play app locally or on GitHub, you can skip this step and go straight to the [Deploy Play App to Railway](#deploy-play-app-to-railway).
```
Comment (brody192) [inline src/docs/guides/play.md:18]: the hash should be `#deploy-the-play-app-to-railway`

File: src/docs/guides/play.md [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+
+Railway offers multiple ways to deploy your Scala app, depending on your setup and preference. 
+
+### One-Click Deploy from a Template
+
+If you’re looking for the fastest way to get started, the one-click deploy option is ideal.
+
+Click the button below to begin:
+
+[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/new/template/my9q_q)
```
Comment (brody192) [inline src/docs/guides/play.md:217]: How about setting a service icon on the `Scala Play` service?

File: src/docs/guides/play.md (hunk 1 above)
Comment (unicodeveloper) [reply src/docs/guides/play.md:20]: Nice catch!

File: src/docs/guides/play.md (hunk 2 above)
Comment (unicodeveloper) [reply src/docs/guides/play.md:18]: Good catch!

File: src/docs/guides/play.md (hunk 3 above)
Comment (unicodeveloper) [reply src/docs/guides/play.md:217]: A service icon? How does that work?

Comment (unicodeveloper) [reply src/docs/guides/play.md:217]: Oh wow, I just figured how to do that. Never knew that was an option in the first place.

Comment (unicodeveloper) [reply src/docs/guides/play.md:217]: I just did that. Very smooth!

File: src/docs/guides/play.md (hunk 1 above)
Comment (unicodeveloper) [reply src/docs/guides/play.md]: Fixed.

**Pull request merged**